### PR TITLE
fix(core-lib): fix error response format of spaceConnector

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/cost-explorer/budget/shared/budget-form/BudgetForm.vue [hunk 1]
```diff
@@ -48,8 +56,19 @@ const createBudget = async () => {
...
+    } catch (e:any) {
+        const ALREADY_EXIST_ERROR_CODE = 'ERROR_BUDGET_ALREADY_EXIST';
+        if (e.code === ALREADY_EXIST_ERROR_CODE) {
+            const selectedProjectOrProjectGroupName = state.baseInfo.project_group_id
+                ? state.projectGroup[state.baseInfo.project_group_id]?.label
+                : state.project[state.baseInfo.project_id]?.label;
+            ErrorHandler.handleRequestError(e, i18n.t('BILLING.COST_MANAGEMENT.BUDGET.ALT_E_CREATE_BUDGET_ALREADY_EXIST', {
+                dataSourceName: state.dataSource[state.baseInfo.data_source_id]?.label,
+                name: selectedProjectOrProjectGroupName,
+            }));
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/budget/shared/budget-form/BudgetForm.vue:68]: praise: 😮

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/cost-explorer/budget/shared/budget-form/BudgetForm.vue [hunk 2]
```diff
@@ -69,6 +88,15 @@ const handleChangeAmountPlanning = (amountPlanInfo: BudgetAmountPlanInfo, isVali
...
 };
+
+// LOAD REFERENCE STORE
+(async () => {
+    await Promise.allSettled([
+        allReferenceStore.load('project'),
+        allReferenceStore.load('projectGroup'),
+        allReferenceStore.load('costDataSource'),
+    ]);
+})();
```
Comment (piggggggggy) [inline apps/web/src/services/cost-explorer/budget/shared/budget-form/BudgetForm.vue:99]: Question: really need this? I knew `getters` already has `load` process.

**Review by piggggggggy — APPROVED** [review]
So~ clear! LGTM!! I left a comment!

**Pull request merged**